eng/pipelines/coreclr/templates/build-perf-maui-apps.yml

```diff
@@ -112,7 +112,7 @@ steps:
 
   # There is a global.json in the runtime repo, but it sets the version to 7.xxx preview. This creates a local global.json to allow the running of 6.0 version.
   - script: |
-      echo '{ "sdk": { "version": "6.0.300", "rollForward": "latestFeature" }}' > ./global.json
+      echo '' > ./global.json 
```

Comment:
is there any need to have the global.json at all if it is empty? Or are you just leaving it here in case we add the JSON back again?

Reply:
In this case, there is a higher level global.json that would be used keeping us from running the sdk version we downloaded (only net7.0+). So keeping the empty global.json file allows us to run the version we downloaded.